kf_lib_data_ingest/etl/transform/auto.py

```diff
@@ -0,0 +1,35 @@
+"""
+Module for 'auto' transforming source data into target service entities.
+Auto transformation does not require guidance from the user on how to merge
+source data tables.
+"""
+import logging
+
+from kf_lib_data_ingest.etl.transform.standard_model.model import StandardModel
+
+
+class AutoTransformer():
+    def __init__(self, target_api_config):
+        self.logger = logging.getLogger(type(self).__name__)
```

Comment:
Not for this PR, but it looks like we have a couple of these invocations in the code. Maybe we should make a default logger mixin?